docker-compose.yml

```diff
@@ -15,6 +15,11 @@ services:
         environment:
             API_URI: http://web:1080
             APP_SECRET:
+    pattern_library:
+        image: ${DOCKER_NAMESPACE}/pattern-library:${REVISION_PATTERN_LIBRARY}
```

Comment:
at the moment, we cannot rely on `REVISION_PATTERN_LIBRARY` to exist on the stateful `.env` of the `unstable` environment, so the `latest` default should be used here too